src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const daySum = days * 40;
```

Comment:
The cost per day should be defined as a constant to avoid magic numbers and make changes easier in the future.